balena/models/device.py

```diff
@@ -1032,8 +1032,38 @@ def set_to_release(self, uuid, commit_id):
 
         """
 
+        device = self.get(uuid)
+        query = {
+            'commit': commit_id,
```

Comment:
How about also supporting an empty `commit_id`, so that we can use this method for unpinning a device?
```
balena.models.device.set_to_release('49b2a76b7f188c1d6f781e67c8f34adb4a7bfd2eec3f91d40b1efb75fe413d')
```

Reply:
@thgreasi noted, will add this improvement in next PR